src/gui/addnewtorrentdialog.cpp

```diff
@@ -263,12 +263,12 @@ bool AddNewTorrentDialog::loadTorrent(const QString &torrentPath)
         BitTorrent::TorrentHandle *const torrent = BitTorrent::Session::instance()->findTorrent(m_hash);
         if (torrent) {
             if (torrent->isPrivate() || m_torrentInfo.isPrivate()) {
-                MessageBoxRaised::critical(this, tr("Already in download list"), tr("Torrent is already in download list. Trackers weren't merged because it is a private torrent."), QMessageBox::Ok);
+                MessageBoxRaised::critical(this, tr("Already in download list"), tr("Torrent '%1' is already in download list. Trackers weren't merged because it is a private torrent.").arg(torrent->name()), QMessageBox::Ok);
```

Comment:
@Chocobo1:  So long as those single-quote marks around the torrent-name argument actually display in the GUI, the English syntax is correct.  :}

Reply:
Thanks! good to know.